plugin/storage/cassandra/schema/docker.sh

```diff
@@ -3,7 +3,7 @@
 # This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
 # that allows installing Jaeger keyspace and schema without installing cqlsh.
 
-CQLSH=/opt/apache-cassandra-3.0.12/bin/cqlsh
+CQLSH=/usr/bin/cqlsh
```

Comment:
This is very specific to the base image that is being used. This script is not meant to be used anywhere else, so, I see little value in making this configurable.